src/frontends/leaflet.ts

```diff
@@ -273,7 +277,7 @@ const leafletLayer = (options: LeafletLayerOptions = {}): unknown => {
 
     // a primitive way to check the features at a certain point.
     // it does not support hover states, cursor changes, or changing the style of the selected feature,
-    // so is only appropriate for debuggging or very basic use cases.
+    // so is only appropriate for debugging or very basic use cases.
```

Comment:
Side note: `queryTileFeaturesDebug()` has proven to be extremely useful for me with on-hover tooltips to gain access to the underlying attributes. 👍